tests/matrix_free/matrix_vector_rt_01.output

```diff
@@ -4,50 +4,50 @@ DEAL:2d::Number of cells: 16
 DEAL:2d::Number of degrees of freedom: 144
 DEAL:2d::
 DEAL:2d::Testing Values 
-DEAL:2d::Norm of difference: 6.05894e-16
+DEAL:2d::Norm of difference: 2.61453e-15
```

Comment:
You would not have needed to update this file (I assume the test was passing also without this change). The reason is that numerical values printed here are compared for a relative tolerance of 1e-6 and an absolute tolerance of 1e-8 against the reference with the `numdiff` tool, so this one here (and all other changes you make in the file) would have been OK already. But it does not harm, either.

Reply:
Oh okay! I'm pretty sure it failed localy for me, otherwise I'd never change it. Not sure why then

Reply:
Oh yes that's what happened! I tried printing the L2-norm of the solution but saw that the output changed from time to time so I changed it back. 👍